config/testdata/config.yaml

```diff
@@ -0,0 +1,57 @@
+library:
+  log:
+    format: text
+    level: 3
+    caller: true
+  profiling: true
+server:
+  adminServer:
+    contextTimeout: 2s
+    http:
+      basePath: /admintest
+      readTimeout: 1s
+      writeTimeout: 3s
+      common:
+        hostName: ""
+        port: 3333
+genCode:
+  handlers:
+    getV1BffFenergoAPIDocs:
```

Comment:
Remove any unnecessary config: `getV1StatementOfPositionCustomer` isn't used outside this file.